src/components/WidgetContainer/widget-container.test.jsx

```diff
@@ -0,0 +1,6 @@
+import { render } from "@testing-library/react";
+import WidgetContainer from "./widget-container";
+
+test("renders WidgetContainer component", () => {
+  render(<WidgetContainer />);
```

Comment:
After rendering, you should verify something; I'd recommend rendering the container, then clicking the button, then verify that the container is open and visible. You may have to tell the test to wait for it to be visible if you add any kind of animation or something like that